CHANGELOG.md

```diff
@@ -1,23 +1,89 @@
 ([Français](#journal-des-modifications))
-
+# Changelog
 ## v1.0.0
 
-Released on: 2025-09-TBD
+Released on: 2025-09-29
+
+### :rocket: CSS Shortcuts v1.0.0 is now live!
+
+CSS Shortcuts is a CSS utility framework built to match GC Design System styles and Canada.ca standards. It provides CSS classes with pre-defined design elements, like colour, spacing, and typography.
+
+Apply these classes to your HTML when you need custom styles for page layouts or elements. Read our new [guidance section](https://design-system.alpha.canada.ca/en/css-shortcuts/) on the website for technical documentation (coming soon!).
 
-### :rocket: New features
+#### Update from `@cdssnc/gcds-utility` to `@gcds-core/css-shortcuts`
+This product has previously been released as an alpha version named GCDS Utility Framework (`@cdssnc/gcds-utility`). We have republished it under a new name, a new home (it’s now under `@gcds-core`), with full technical documentation which marks its official product release: `@gcds-core/css-shortcuts`.
 
-- First release of GC Design System CSS Shortcuts. 
+As part of this change, the version number has been reset to v1.0.0, so you’ll need to update your references to continue receiving future improvements. Here’s a guide to help with update your code:
 
-Note: CSS Shortcuts was previous released as GCDS Utility, or GCDS Utility Framework (gcds-utility). An archived version of the CHANGELOG can be found [here](archived/CHANGELOG-UTILITY.md)
+**Update the stylesheet:**
+1. Our page templates use the CSS Shortcuts (previously `gcds-utility`) stylesheet by default. Replace your existing stylesheet link with the following. Note the new path, version number, and filename.
```

Comment:
How do you feel about removing the `1.` here since there aren't any other numbers?

```suggestion
Our page templates use the CSS Shortcuts (previously `gcds-utility`) stylesheet by default. Replace your existing stylesheet link with the following. Note the new path, version number, and filename.
```